src/createStore.ts

```diff
@@ -39,35 +39,54 @@ import { kindOf } from './utils/kindOf'
  * @returns A Redux store that lets you read the state, dispatch actions
  * and subscribe to changes.
  */
+
+// 3 overloaded functions
```

Comment:
I posted this question in the teams channel, but do you think we should conver these comments into JSDocs format? They would then become accessible via intellisense and we could run other auto-documentation libraries against them. Could be useful in the future.

Reply:
I am open to it but I am not familiar with it.